subprojects/emcee/worker/src/main/kotlin/com/avito/emcee/worker/internal/di/WorkerDI.kt

```diff
@@ -0,0 +1,68 @@
+package com.avito.emcee.worker.internal.di
+
+import com.avito.emcee.worker.Config
+import com.avito.emcee.worker.WorkerQueueApi.Companion.createWorkerQueueApi
+import com.avito.emcee.worker.internal.TestJobProducer
+import com.avito.emcee.worker.internal.TestJobProducerImpl
+import com.avito.emcee.worker.internal.consumer.FakeTestJobConsumer
+import com.avito.emcee.worker.internal.consumer.TestJobConsumer
+import com.avito.emcee.worker.internal.identifier.HostnameWorkerIdProvider
+import com.avito.emcee.worker.internal.identifier.WorkerIdProvider
+import com.avito.emcee.worker.internal.networking.SocketAddressResolver
+import com.avito.emcee.worker.internal.rest.HttpServer
+import com.avito.emcee.worker.internal.rest.handler.HealthCheckRequestHandler
+import com.avito.emcee.worker.internal.rest.handler.ProcessingBucketsRequestHandler
+import com.avito.emcee.worker.internal.storage.ProcessingBucketsStorage
+import com.avito.emcee.worker.internal.storage.SingleElementProcessingBucketsStorage
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import kotlin.time.ExperimentalTime
+
+@ExperimentalTime
+internal class WorkerDI(
```

Comment:
dagger?

Reply:
Let's stay without dagger. Maybe in the future